src/modules/MouseWithoutBorders/App/Form/frmMatrix.cs

```diff
@@ -797,6 +797,14 @@ private void FrmMatrix_Load(object sender, EventArgs e)
                 checkBoxHideLogo.Enabled = false;
             }
 
+            // Note(@htcfreek): Disable checkboxes of settings that we don't support in the PowerToys implementation
+            checkBoxDisableCAD.Enabled = false;
+            checkBoxDisableCAD.Text = checkBoxDisableCAD.Text + " [Unsupported!]";
```

Comment:
Maybe we want only the text `[Setting is not supported!]` or `[Setting has been removed!]` without the original name.

Reply:
I guess it's fine as it is. This is mostly about having this available for troubleshooting.